config/vufind/GetThis.yaml

```
@@ -0,0 +1,96 @@
+---
+# This file contains the settings for the GetThis feature
+
+# Either to display the copy number (next to the call number), default false
+showCopyNumber: true
+
+templates:
+  # Elements here correspond to a template file within templates/get-this with .phtml being stripped from the end
+  # The order changes the order templates are displayed
+  # Elements parameter, all parameters are optional :
+  #   enabled: true/false, whether to enable the template
+  #   view_variables: array of extra variables made available to the templates.
+  #     It can be used with placeholders if available; see example below.
+  #   condition_function: a function within GetThisLoader returning a boolean to whether display or not the template
+  #     The result of the function can be negated if prefixed with "!"
+  #     note : If the function does not exist, the condition will evaluate to false
+  #   conditions: if condition_function is limiting for you, conditions replace it to give more flexibility :
+  #     This element defines a set of logical conditions.
+  #     The structure is recursive, allowing nested combinations of AND/OR logic.
+  #     At each level, conditions is a list of entries.
+  #     If any entry is true, the whole list is considered true (logical OR).
+  #     Entries can be:
+  #       condition_function: evaluates a single condition.
+  #       conditions: e nested conditions block used to combine conditions.
+  #       The string "and": acts as a logical AND between adjacent conditions.
+  #
+  #   Example; Values surrounded with brackets are placeholders serving as example :
+  #     {template_name}:
+  #       conditions:
+  #         - conditions:
+  #             - conditions:
+  #                 - condition_function: {test_function1}
+  #                 - and
+  #                 - condition_function: {test_function2}
+  #             - condition_function: {test_function3}
+  #         - condition_function: {test_function4}
+  #       view_variables:
+  #         - {my_link}: {https://your_lib.edu}
+  #         - {my_link_with_placeholder}: {https://mylib.edu/%%location%%/open_hours}
+
+  holdings:
+    condition_function: showHoldings
+
+  biblio-info:
+    enabled: true
+
+  micro-form:
+    condition_function: showMicroForm
+    view_variables:
+      open_hours: https://your_lib.edu/hours
+
+  message:
+    condition_function: showMessage
+
+  staff-office-delivery:
+    condition_function: showStaffDelivery
+    view_variables:
+      # Has the following placeholders available:
+      #   %%item%% : Item record
+      #   %%location%%: Location of the item
+      #   %%callNum%%: Call number
+      #   %%title%%: Title
+      #   %%author%%: Author
+      #   %%isbn%%: ISBN
+      general_link: https://your_lib.msu.edu/facstaffdelivery
```

Stray msu slipped through here:

```suggestion
      general_link: https://your_lib.edu/facstaffdelivery
```